pkg/pb/evaluator.pb.go

```diff
@@ -1,288 +1,160 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
 // Code generated by protoc-gen-go. DO NOT EDIT.
-// versions:
-// 	protoc-gen-go v1.25.0
-// 	protoc        v3.10.1
```

Comment:
Look here @Laremere. When i run `make all-protos` to generate the changes from proto files, it deletes the licensing here. How should we proceed with this?? Do i leave it like it is or what?? Little help, please

Reply:
If that's what the commands do - let's leave it like that.

Reply:
ok, great!